src/Sulu/Bundle/TrashBundle/Resources/config/lists/trash_items.xml

```diff
@@ -31,7 +31,7 @@
 
     <properties>
         <property
-            name="resourceKey"
+            name="resourceType"
```

Comment:
is there a specific reason for this change? 🙂

Reply:
I accidently changed it in #6326. Thought it was an accident but it is required as the `resourceType` is the item shown in the list e.g.: `Page` and the resourceKey (hardcoded added in the template) is the untranslated `resourceKey`. Currently the FormConfiguration.